data/jibauni/jibauni culture conversations.txt

```diff
@@ -0,0 +1,248 @@
+				decline
+			label runinto
+			`	As you stand where you are, the Jibauni come racing your way, knocking you down as the chase continues. Once all of the soldiers pass, a bystander helps you up and brushes you off, telling you that scenes like this are fairly frequent, happening either when a Jibauni is accused of stealing... or does something people of power dislike.`
+				decline
+			label question
+			`	All of the Jibauni keep running past you, intent on what they are doing. After the traces are gone, a bystander tells you that scenes like this are fairly frequent, when a Jibauni is accused of stealing... or doing something the Governor does not like.`
+				decline
+
+mission "Jibauni Culture: Hajay Intersystem"
+	minor
+	source "Kanlil"
+	to offer
+		has "language: Jibauni"
+		random < 1
+	on offer
+		conversation
+			`A Quyan lands shortly after <ship> touches down, but it seems to have more Jibauni disembarking than any normal Quyan could have. Do you want to go investigate?`
+			choice
+				`	(Why not.)`
+				`	(No.)`
+					goto no
+			`	You walk up to the Quyan, whose name you notice is "C. S. Jandon". Indeed, the number of passengers is more than usual.`
+			choice
+				`	"How do you fit so many Jibauni on board?"`
+				`	(Never mind. Get out of here before the captain notices you.)`
+					goto no
+			`	"Oh," the captain says. "I removed my hyperdrive. Simple, easy, and it lets you fit four more passengers for nothing."`
+			choice
+				`	"Why would you do that?"`
+				`	"I see."`
+			`	"I do it because the peaceful captain life is good enough for me. I don't need to go gallivanting around to strange places. I've been in Hajay my whole life, and enjoyed every bit of it. May I ask you a question?"`
+			choice
+				`	"Sure."`
+					goto herquestion
+				`	"If you want."`
+					goto herquestion
+				`	"No thanks."`
+					goto nothanksship
+			label herquestion
+			`	"What is the name of your ship, <first>?"`
+			choice
+				`	"<ship>."`
+					goto toldship
+				`	"Wait. How do you know my name?"`
+					goto namequestion
+				`	"I thought you would know that I am Captain <first> <last> of the <ship>."`
+					goto thoughtyouknew
+			label toldship
+			`	"Okay. You probably already can see mine," the Jibauni winks at you.`
+				goto whoyou
+			label namequestion
+			`	"You are the only living non-Jibauni anywhere in our space," the Jibauni reminds you. "Everyone knows who you are. Will you answer my question?"`
+			choice
+				`	"<ship>."`
+					goto toldship
+				`	"I thought you would know that I am Captain <first> <last> of the <ship>."`
+					goto thoughtyouknew
+			label thoughtyouknew
+			`	"Nope," the Jibauni replies merrily. "We common Jibauni only know your first and last names."`
+				goto whoyou
+			label whoyou
+			`	"What is your name?" you ask. "I am surprised you would want to know," is the answer. "But I am Ru-win, daughter of Alson and Weni. At any rate, I have some business to attend to, so goodbye."`
```

Comment:
Took a stab at adding in the naming option you mentioned. Still some fill-in-the-blanks left for name meaning though.
```suggestion
			`	"What is your name?" you ask. "I am surprised you would want to know," is the answer. "But I am Ru-win, daughter of Alson and Weni."`
			choice
				`	"Thanks. Well, I need to get going."`
					goto finish
				`	"That's a nice name, what does it mean?"`
			`	"Ah, my name means blah-blah-blah. The 'Ru' means xyz, and names ending in 'n' are traditionally used for female Jibauni. Traditional male names usually end in 'a.' At any rate, I have some business to attend to, so goodbye."`
```

Reply:
```suggestion
			`	"What is your name?" you ask. "I am surprised you would want to know," is the answer. "But I am Ru-win, daughter of Alson and Weni."`
			choice
				`	"Thanks. Well, I need to get going."`
					goto finish
				`	"That's a nice name, what does it mean?"`
			`	"Ah, my name means 'The pilot'. The 'Ru' means 'pilot', and names ending in consonants are traditionally used for female Jibauni. Traditional male names usually end in vowels. At any rate, I have some business to attend to, so goodbye."`
```